eng/scripts/Language-Settings.ps1

```diff
@@ -154,6 +154,7 @@ function Get-AllPackageInfoFromRepo($filterString)
     $searchPaths = ResolveSearchPaths $filterString
 
     foreach ($searchPath in $searchPaths) {
+      Write-Host "Searching for go modules in $searchPath"
```

Comment:
How noisy does this make discovery? In general I've tried to keep all logging to verbose only in the package discovery because it can be noisy when doing things like prepare-release script runs for example.

Reply:
Ah you know, good point.

Reply:
I'm making a change for proxy and non-proxy in the pullrequest builds, will remove this then.